_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -133,8 +133,9 @@ test.describe('Popular Repositories', () => {
     await test.step('Shared EPEL repository cannot be edited or deleted', async () => {
       const row = await getRowByNameOrUrl(page, repoName10);
       await row.getByLabel('Kebab toggle').click();
-      await expect(page.getByRole('menuitem', { name: 'Edit' })).not.toBeVisible();
-      await expect(page.getByRole('menuitem', { name: 'Delete' })).not.toBeVisible();
+      await expect(page.getByRole('menu')).toBeVisible();
+      await expect(page.getByRole('menuitem', { name: 'View all snapshots' })).toBeVisible();
```

Comment:
we don't snapshot the epel repo in CI since that would take way too long, so we should only see one item (`No snapshots yet`) in the menu